recommender_system/README.en.md

```diff
@@ -83,11 +83,6 @@ We use the [MovieLens ml-1m](http://files.grouplens.org/datasets/movielens/ml-1m
 
 `paddle.v2.datasets` package encapsulates multiple public datasets, including `cifar`, `imdb`, `mnist`, `moivelens` and `wmt14`, etc. There's no need for us to manually download and preprocess `MovieLens` dataset.
 
-```python
-# Run this block to show dataset's documentation
-help(paddle.v2.dataset.movielens)
```

Comment:
paddle need to be imported before using `paddle.v2.dataset.movielens`, but the introduction for paddle import and init is below. And importing paddle twice will cause jupytor error. So I am removing this line.